cmd/influx/main_test.go

```diff
@@ -31,6 +31,42 @@ func TestParseCommand_CommandsExist(t *testing.T) {
 	}
 }
 
+func TestParseCommand_CommandsSamePrefix(t *testing.T) {
+	t.Parallel()
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var data client.Response
+		w.WriteHeader(http.StatusNoContent)
+		_ = json.NewEncoder(w).Encode(data)
+	}))
+	defer ts.Close()
+
+	u, _ := url.Parse(ts.URL)
+	config := client.Config{URL: *u}
+	c, err := client.NewClient(config)
+	if err != nil {
+		t.Fatalf("unexpected error.  expected %v, actual %v", nil, err)
+	}
+	m := main.CommandLine{Client: c}
+
+	tests := []struct {
+		cmd string
+	}{
+		{cmd: "use db"},
```

Comment:
It would be nice in the test to understand if these commands selected the correct "code branch".  I understand it isn't possible with the way the current method is written.  I'm only bringing it up as it may be something to think about when you refactor the cli main package going forward.